source/extensions/filters/network/redis_proxy/config.h

```diff
@@ -13,11 +15,31 @@ namespace Extensions {
 namespace NetworkFilters {
 namespace RedisProxy {
 
+class ProtocolOptionsConfigImpl : public Upstream::ProtocolOptionsConfig {
+public:
+  ProtocolOptionsConfigImpl(
+      const envoy::config::filter::network::redis_proxy::v2::RedisProtocolOptions& proto_config)
+      : auth_password_(proto_config.auth_password()) {}
+
+  std::string auth_password(Api::Api& api) const {
+    return Config::DataSource::read(auth_password_, true, api);
+  }
+
+  const envoy::api::v2::core::DataSource& auth_password_datasource() const {
+    return auth_password_;
+  }
+
+private:
+  envoy::api::v2::core::DataSource auth_password_;
```

Comment:
nit: const here and in similar places if possible.

Reply:
ok